src/store.js

```diff
@@ -12,7 +12,12 @@ export const initialState = {
       lastChecked: Date.now(),
     },
     editor: { // The state of the editor
-      currentResource: undefined,
+      currentPath: undefined,
+      diacritics: {
+        show: false,
+        targetRef: undefined,
```

Comment:
Can we avoid storing a ref and function in state? We don't do this anywhere else.

Reply:
Done, now passing reduxState and no longer trying to set the language code based on the character set.